codebuild.go

```diff
@@ -1513,7 +1513,9 @@ func (p *CodeBuilder) refMember(typ types.Type, name string, argVal ast.Expr, sr
 }
 
 func (p *CodeBuilder) fieldRef(x ast.Expr, o *types.Struct, name string, src ast.Node) bool {
-	for i, n := 0, o.NumFields(); i < n; i++ {
+	n := o.NumFields()
+	embed := make([]*types.Var, 0, n)
```

Comment:
don't need prealloc (because embed fields are not popular)

Reply:
fixed